src/diffusers/models/embeddings.py

```diff
@@ -878,3 +878,28 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
         latents = self.proj_out(latents)
         return self.norm_out(latents)
+
+
+class MultiIPAdapterImageProjection(nn.Module):
+    def __init__(self, IPAdapterImageProjectionLayers: Union[List[nn.Module], Tuple[nn.Module]]):
+        super().__init__()
+        self.ImageProjectionLayers = nn.ModuleList(IPAdapterImageProjectionLayers)
```

Comment:
```suggestion
        self.image_projection_layers = nn.ModuleList(IPAdapterImageProjectionLayers)
```

Reply:
No camel case names for weights please

Reply:
@patrickvonplaten why do we not use camel case names for weights?